### ISSUE TITLE
Postal Code?

### ISSUE THREAD

**Issue by chobo2** [medipass/react-credit-card-input#41]
Hi

Is only USA supported? How about the countries that have postal codes and not a zip code?

**Comment by jxom** [comment]
Yes, only USA is supported for the moment. I am currently finishing up a new package for using payment inputs with React, which I hope to publish soon! 

The new package will have more flexibility for global ZIP codes.

If you're interested, I'm planning to add a migration guide to migrate from react-credit-card-input to the new package - don't worry, it will be a small migration. 

I'll comment here again once it's published. :)

**Comment by jxom** [comment]
We have just recently published a new library, [React Payment Inputs](https://github.com/medipass/react-payment-inputs) which is a rewrite of react-credit-card-input, and fixes a lot of issues that were raised in this repository as well as a few extra enhancements.

Your issue that you raised here should be fixed in React Payment Inputs - however, if you wish to still use react-credit-card-inputs, I'll try find time to fix this issue in the future.
